test/images/agnhost/connect/connect.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package connect
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"syscall"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+// CmdConnect is used by agnhost Cobra.
+var CmdConnect = &cobra.Command{
+	Use:   "connect",
```

Comment:
nit: the `Use` should show what the arg it expects is. Only the first word is used as the command name, the rest is for help messages.

```
Use:   "connect [tcp destination]",